src/Emulator/Tabs/ConfigureTab/ConfigureTab.js

```diff
@@ -0,0 +1,136 @@
+import React, { useState, useRef } from 'react';
+import './ConfigureTab.css';
+import DeviceContainer from './DeviceContainer/DeviceContainer';
+
+const MAX_DEVICES = 10;
+
+// Sample responses from currently non-existent API
+// These represent the starting config for a lesson
+const routerConfig = [
+  {
+    name: 'r1',
+    connections: ['s1', 's2'],
+  },
+];
+const switchConfig = [
+  {
+    name: 's1',
+    connections: ['r1', 'h1', 'h2', 'h3'],
+  },
+  {
+    name: 's2',
+    connections: [],
+  },
+];
+const hostConfig = [
+  {
+    name: 'h1',
+    connections: ['s1'],
+  },
+  {
+    name: 'h2',
+    connections: ['s1'],
+  },
+  {
+    name: 'h3',
+    connections: ['s1'],
+  },
+  {
+    name: 'h4',
+    connections: [],
+  },
+];
+
+const getNextNumber = (s) => Number(s.slice(1)) + 1;
+const getNextDeviceName = (device) => {
+  const lastDeviceName = device[device.length - 1].name;
+  return `${lastDeviceName[0]}${getNextNumber(lastDeviceName)}`;
+};
+
+const ConfigureTab = ({ status }) => {
+  const [routers, setRouters] = useState(routerConfig);
+  const [switches, setSwitches] = useState(switchConfig);
+  const [hosts, setHosts] = useState(hostConfig);
+
+  const routerScrollRef = useRef(null);
+  const switchScrollRef = useRef(null);
+  const hostScrollRef = useRef(null);
+
+  const scrollDeviceContainer = (ref) => {
```

Comment:
**Some good tests to add to this component**
`The ConfigureTab`
- [ ] should be able to add new devices
- [ ] should limit devices to 10
- [ ] should increment device names `r1, r2, ... rn`

Reply:
[] add device to an empty intial config